### DISCUSSION TITLE
[feature request] Rough timeline for supporting markdown Definition lists ?

### DISCUSSION THREAD

**Discussion by wolfspyre** [GitJournal/GitJournal#937] [Feature Requests]
First I want to say GitJournal looks awesome.

Nicely done!

I wanted to inquire about the liklihood of support for definition lists.

I'm open to alternative mechanisms, but FutureMe™ has found it pretty handy to have the ability to curate, and interlink to  a gaggle of links at the bottom of a document I wrote some time ago.. and has asked CurrentMe™  to continue this habit.

If this is a thing that y'all are planning on implementing (at some point in time) that's fine.... I just wanted to make sure that there wasn't something that's PREVENTING them from being implemented whenever the desire becomes sufficient to merit the effort.

**Comment by vHanda** [comment]
Moving this is discussion as this isn't a bug, and I'd prefer for the issue tracker to just have bugs, and bring it down as much as possible.

Also, to be honest, from your comment, I don't quite understand exactly what you'd like. Could you please give me an example? I see definition lists from here - https://www.markdownguide.org/extended-syntax/#definition-lists . I'm assuming you'd like GitJournal to parse them and render them somehow?

The easiest way to get support for this would be to implement it upstream - https://pub.dev/packages/flutter_markdown -

**Reply by wolfspyre** [reply]
yeah.. I was/am referring to markdown definition lists[^1].

I’ll check in with the upstream markdown package. 

[^1]: they’re quite handy

**Comment by wolfspyre** [comment]
beyond the rendering of them, I figured that it might be worthwhile to maintain a table of them somewhere for display/enhancement/referencing outside of just markdown rendering

**Comment by wolfspyre** [comment]
on the rendering side, they make terms and concepts a whole lot easier to visually represent… 

on the referencing side, i figured it might be viable to store a table of content headings that could be cross-linked…
it’d be rad if  definitionlists could also fall into that category
